.travis.yml

```diff
@@ -1,33 +1,44 @@
-# here we are following this example
-# https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html
-
 notifications:
   email: false
 
-language: python
+language: minimal
+
+matrix:
+  fast_finish: true
+  include:
+    - name: "python-3.6"
+      env: PY=3.6
+    - name: "python-3.7"
+      env: PY=3.7
+    - name: "tarball"
+      env: PY=3.7
 
 before_install:
-  - "pip install -U pip"
```

Comment:
No need to update `pip`, or to have pip at all, when using conda.
Note that you still have `conda`'s `pip` installed, that is the system `pip`.